be/src/storage/persistent_index.cpp

```diff
@@ -3284,12 +3283,16 @@ bool PersistentIndex::_can_dump_directly() {
     return _dump_bound() <= config::l0_snapshot_size;
 }
 
-bool PersistentIndex::_need_flush_advance() {
+bool PersistentIndex::_l0_is_full() {
     const auto l0_mem_size = _l0->memory_usage();
-    uint64_t l1_file_size = _has_l1 ? _l1_vec[0]->file_size() : 0;
-    bool flush_advance = (l1_file_size != 0) ? l0_mem_size * config::l0_l1_merge_ratio > l1_file_size
-                                             : l0_mem_size > config::l0_max_mem_usage;
-    return flush_advance;
+    auto manager = StorageEngine::instance()->update_manager();
+    return l0_mem_size >= config::l0_max_mem_usage ||
```

Comment:
Is the 100MB too small?

Reply:
Tablet's memtable default size is also 100MB, maybe enough?